### ISSUE TITLE
Geos CAPI tests are not run on travis-ci

### ISSUE THREAD

**Issue by tneems** [rgeo/rgeo#118]
A bunch of the tests/assertions are not being executed on travis-ci. I believe these are the Geos CAPI tests.
```
Finished in 1.736912625 seconds.
--------------------------------------------------------------------------------
1062 tests, 3099 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
--------------------------------------------------------------------------------
611.43 tests/s, 1784.20 assertions/s
```

On my local machine.
```
Finished in 1.332574 seconds.
--------------------------------------------------------------------------------
1273 tests, 3684 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
--------------------------------------------------------------------------------
955.29 tests/s, 2764.57 assertions/s
```

**Comment by deivid-rodriguez** [comment]
This should be fixed by #120

Now I get 

```
--------------------------------------------------------------------------------
1327 tests, 3756 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
--------------------------------------------------------------------------------
1321.74 tests/s, 3741.10 assertions/s
```

in both travis and my local machine.

**Comment by tneems** [comment]
Fixed with https://github.com/rgeo/rgeo/pull/120.
